aries-site/src/components/feedback/Feedback.js

```diff
@@ -0,0 +1,124 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  Form,
+  Heading,
+  Layer,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { FormClose } from 'grommet-icons';
+
+export const Feedback = ({
+  children,
+  modal,
+  onChange,
+  onClose,
+  onEsc,
+  onSubmit,
+  onReset,
+  show,
+  title,
+  value,
+  isSucessful,
+}) => {
+  const size = useContext(ResponsiveContext);
```

Comment:
Moving towards `breakpoint` instead of `size` as best practice. More accurate representation as to what it is.

```suggestion
  const breakpoint = useContext(ResponsiveContext);
```

Reply:
changed thanks for the reminder